e2e/app/page/notebook-page.ts

```diff
@@ -297,25 +299,50 @@ export default class NotebookPage extends NotebookFrame {
       .catch(() => false);
   }
 
+  async isNoConnection(): Promise<boolean> {
+    const kernelStatus = await this.getKernelStatus();
+    if (kernelStatus === KernelStatus.NoConnection) {
```

Comment:
how about simply `return (kernelStatus === KernelStatus.NoConnection)` ?

Reply:
👍🏻 It's better. I should have thought that.